README.md

```diff
@@ -11,71 +11,73 @@
 
 ## Goals
 
-Syncthing is a **continuous file synchronization program**. It synchronizes
-files between two or more computers. We strive to fulfill the goals below.
+Syncthing is a **continuous file synchronization program**.  
+It synchronizes files between two (or more) computers. 
+We strive to fulfill the goals below.
 The goals are listed in order of importance, the most important one being
-the first. This is the summary version of the goal list - for more
+the first. This is the summary version of the goal list &mdash; for more
 commentary, see the full [Goals document][13].
 
 Syncthing should be:
 
-1. **Safe From Data Loss**
+1. **Safe from Data Loss**
```

Comment:
There are different valid styles of Title Case, the previous variant included.  I see no benefit in touching this, or do you have a concise reasoning why it needs to be done?